encord/metadata_schema.py

```diff
@@ -0,0 +1,176 @@
+from enum import Enum
+from typing import Annotated, Literal, Sequence
+
+from pydantic import BaseModel, Field, RootModel
+
+from encord.http.v2.api_client import ApiClient
+
+__all__ = ["MetadataSchema", "MetadataSchemaError", "ClientMetadataSchemaTypeEmbedding"]
+
+
+class _NumberFormat(Enum):
+    F32 = "F32"
+    F64 = "F64"
+
+
+class _ClientMetadataSchemaTypeNumber(BaseModel):
```

Comment:
SDK has to use BaseDTO instead of BaseModel but should go through okay. Be careful around RootModels